prog/vm/github_runner.rb

```diff
@@ -405,6 +405,14 @@ def collect_final_telemetry
       }
       Clog.emit("Remaining DockerHub rate limits") { {dockerhub_rate_limits:} }
     end
+
+    proxy_log_command = <<~COMMAND
```

Comment:
nit: since we don't seem to reuse the command, putting the string literal in the `.cmd` args might look cleaner